### PR TITLE
Add autos to master

### PR DESCRIPTION
**Pull request by MatthewRing**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -85,9 +104,27 @@ public void disabledPeriodic() {
    */
   @Override
   public void autonomousInit() {
-    m_autonomousCommand = m_robotContainer.getAutonomousCommand();
 
-    // schedule the autonomous command (example)
+    // Schedule auto command chosen on Shuffleboard.
+    switch (autoChooser.getSelected()) {
+
+    case 1:
```
Comment (JBakks) [inline src/main/java/frc/robot/Robot.java:111]: Please fix your indentation, case1 should be indented one more.

Comment (JBakks) [reply src/main/java/frc/robot/Robot.java:111]: not just case1 but all cases

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -84,18 +84,20 @@ private void configureButtonBindings() {
...
+        // When a button is pressed runs its respective method inside gripperCommand.
         gripperOpen.onTrue(new GripperCommand(true));
         gripperClose.onTrue(new GripperCommand(false));
 
     }
 
+    /* 
     public Command getAutonomousCommand() {
-        return new AutoPath(s_Swerve);
+        return new Auto4();
```
Comment (JBakks) [inline src/main/java/frc/robot/RobotContainer.java:99]: You're returning Auto4 here, wont that always run auto 4?

File: src/main/java/frc/robot/autos/Auto3.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import frc.robot.commands.ZeroArmCommand;
+import frc.robot.commands.autoCommands.AutoPath;
+
+public class Auto3 extends SequentialCommandGroup {
+
+    public Auto3() {
+
+        // Play command sequence.
+        addCommands(new GripperCommand(false),
+                new ZeroArmCommand(),
```
Comment (JBakks) [inline src/main/java/frc/robot/autos/Auto3.java:14]: Why zero the arm here, shouldn't it already be at zero

File: src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -85,9 +104,27 @@ public void disabledPeriodic() {
    */
   @Override
   public void autonomousInit() {
-    m_autonomousCommand = m_robotContainer.getAutonomousCommand();
 
-    // schedule the autonomous command (example)
+    // Schedule auto command chosen on Shuffleboard.
+    switch (autoChooser.getSelected()) {
```
Comment (JBakks) [inline src/main/java/frc/robot/Robot.java:109]: Indentation needs to be fixed here, all cases should be one more in

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (JBakks) [inline src/main/java/frc/robot/RobotContainer.java]: Why is auto 4 being returned here?

File: src/main/java/frc/robot/autos/Auto3.java (hunk 3 above)
Comment (JBakks) [inline src/main/java/frc/robot/autos/Auto3.java]: Why are you zeroing the arm, shouldn't it already be at zero?

File: src/main/java/frc/robot/commands/autoCommands/AutoPath.java [hunk 5]
```diff
@@ -12,51 +12,42 @@
...
-    public AutoPath(Swerve driveSubsystem) {
-        // This will load the file "FullAuto.path" and generate it with a max velocity
-        // of 4 m/s and a max acceleration of 3 m/s^2
-        // for every path in the group
+
+    Swerve driveSubsystem = Swerve.getInstance();
+
+    // Creates path with default velocity settings.
+    public AutoPath(String path) {
+        this(path, new PathConstraints(4, 3));
```
Comment (JBakks) [inline src/main/java/frc/robot/commands/autoCommands/AutoPath.java]: Could you make variables in constants for max velocity and acceleration?

**Review by JBakks — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (MatthewRing) [reply src/main/java/frc/robot/RobotContainer.java]: Just leftover from testing that I missed removal of. It isn't needed anymore as everything is being done in robot.java.

File: src/main/java/frc/robot/Constants.java [hunk 6]
```diff
@@ -144,6 +144,8 @@ public static final class AutoConstants { //TODO: The below constants are used i
     /* Constraint for the motion profilied robot angle controller */
     public static final TrapezoidProfile.Constraints kThetaControllerConstraints = new TrapezoidProfile.Constraints(
         kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
+
+    public static final PathConstraints maxVelocity = (4, 3);
```
Comment (JBakks) [inline src/main/java/frc/robot/Constants.java]: could you make velocity and accerlation separate?

for example,
public static final velocity = 4;
public static final acceleration = 3;

File: src/main/java/frc/robot/commands/autoCommands/AutoPath.java [hunk 7]
```diff
@@ -20,7 +20,7 @@ public class AutoPath extends SequentialCommandGroup {
 
     // Creates path with default velocity settings.
     public AutoPath(String path) {
-        this(path, new PathConstraints(4, 3));
+        this(path, new PathConstraints(Constants.AutoConstants.maxVelocity));
```
Comment (JBakks) [inline src/main/java/frc/robot/commands/autoCommands/AutoPath.java]: here you would reference 
Constants.AutoConstants.velocity, Constants.AutoConstants.acceleration
instead

**Review by JBakks — CHANGES_REQUESTED** [review]

**Review by JBakks — APPROVED** [review]

**Pull request merged**